specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2024-03-03/examples/galleryResourceProfileExamples/GalleryInVMAccessControlProfileVersion_Delete.json

```diff
@@ -0,0 +1,30 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2024-03-03",
+    "galleryName": "myGalleryName",
+    "inVMAccessControlProfileName": "myInVMAccessControlProfileName",
+    "inVMAccessControlProfileVersionName": "1.0.0"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Azure-AsyncOperation": "https://management.azure.com/subscriptions/000000-8de3-42e7-b00e-8970003527749a/providers/Microsoft.Compute/locations/westus/capsOperations/0012b61c-2d36-40bc-b7ed-1f0e48757277?api-version=2024-03-03",
+        "Location": "https://management.azure.com/subscriptions/000000-8de3-42e7-b00e-8970003527749a/providers/Microsoft.Compute/locations/westus/capsOperations/0012b61c-2d36-40bc-b7ed-1f0e48757277?monitor=true&api-version=2024-03-03"
+      }
```

Comment:
200 \204 should be sync .. not location headers\ async operations

Reply:
yeah... fixed. will fix other examples before the actual swagger PR